specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/stable/2023-07-20/examples/AzureLargeInstance_ListByResourceGroup.json

```diff
@@ -0,0 +1,106 @@
+{
+  "parameters": {
+    "api-version": "2023-07-20",
+    "subscriptionId": "f0f4887f-d13c-4943-a8ba-d7da28d2a3fd",
+    "resourceGroupName": "myResourceGroup"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/f0f4887f-d13c-4943-a8ba-d7da28d2a3fd/resourceGroups/myResourceGroup/providers/Microsoft.AzureLargeInstance/AzureLargeInstance/myAzureLargeMetalInstance1",
+            "name": "myAzureLargeMetalInstance1",
+            "type": "Microsoft.AzureLargeInstance/AzureLargeInstance",
+            "location": "westus",
+            "tags": {
+              "key": "value"
+            },
+            "systemData": {
+              "createdBy": "user@microsoft.com",
+              "createdByType": "User",
+              "createdAt": "2020-12-20T23:10:22.6828621Z",
+              "lastModifiedBy": "user@microsoft.com",
+              "lastModifiedByType": "User",
+              "lastModifiedAt": "2021-07-21T08:01:22.0000000Z"
+            },
+            "properties": {
+              "azureLargeInstanceId": "23415635-4d7e-41dc-9598-8194f22c24e1",
+              "powerState": "started",
+              "proximityPlacementGroup": "/subscriptions/f0f4887f-d13c-4943-a8ba-d7da28d2a3fd/resourceGroups/myResourceGroup/providers/Microsoft.Compute/proximityPlacementGroups/myplacementgroup",
+              "hwRevision": "Rev 3",
+              "hardwareProfile": {
+                "hardwareType": "Cisco_UCS",
+                "azureLargeInstanceSize": "S72"
+              },
+              "networkProfile": {
+                "networkInterfaces": [
+                  {
+                    "ipAddress": "100.100.100.100"
+                  }
+                ],
+                "circuitId": "/subscriptions/f0f4887f-d13c-4943-a8ba-d7da28d2a3fd/resourceGroups/myResourceGroup/providers/Microsoft.Network/expressRouteCircuit"
+              },
+              "storageProfile": {
+                "nfsIpAddress": "200.200.200.200"
+              },
+              "osProfile": {
+                "computerName": "myComputerName1",
+                "osType": "SUSE",
+                "version": "12 SP1",
+                "sshPublicKey": "AAAAB3NzaC1yc2EAAAABJQAAAQB/nAmOjTmezNUDKYvEeIRf2YnwM9/uUG1d0BYsc8/tRtx+RGi7N2lUbp728MXGwdnL9od4cItzky/zVdLZE2cycOa18xBK9cOWmcKS0A8FYBxEQWJ/q9YVUgZbFKfYGaGQxsER+A0w/fX8ALuk78ktP31K69LcQgxIsl7rNzxsoOQKJ/CIxOGMMxczYTiEoLvQhapFQMs3FL96didKr/QbrfB1WT6s3838SEaXfgZvLef1YB2xmfhbT9OXFE3FXvh2UPBfN+ffE7iiayQf/2XR+8j4N4bW30DiPtOQLGUrH1y5X/rpNZNlWW2+jGIxqZtgWg7lTy3mXy5x836Sj/6L"
```

Comment:
I'm assuming this is a fake key, but is there any way to put a placeholder instead of a key here?

Reply:
I decided to put a string literal like what's used in the microsoft rest api docs - "{ssh-rsa public key}"